src/components/line-chart.js

```diff
@@ -214,6 +232,11 @@ class CoronaChart extends Component {
             <input type="radio" id="log" name="scale" value="log" checked={this.state.scale === 'log'}/>
             <span style={{color: "#AAA"}}>Log</span>
           </span>
+          <span
```

Comment:
I'm thinking of putting a float 'right' on this a then a width 80% on the encapsulating div.

Reply:
<img width="941" alt="image" src="https://user-images.githubusercontent.com/2278040/78939928-4c89f400-7ab5-11ea-98d1-2b31394ef3d6.png">
Something like this